workshops/views.py

```diff
@@ -375,6 +375,61 @@ def person_bulk_add_confirmation(request):
         return render(request, 'workshops/person_bulk_add_results.html',
                       context)
 
+@login_required
+def person_find_duplicates(request):
```

Comment:
On Tue, Mar 31, 2015 at 12:36:28PM -0700, Piotr Banaszkiewicz wrote:
> 4) There's no handling for methods other than `POST` and `GET` (I'm
> nitpicking here.)

To save some lookup, the correct status for unsupported methods is 405
(Method Not Allowed) [1].

[1]: http://tools.ietf.org/html/rfc7231#section-6.5.5